src/app/pages/stats-page/stats-page.component.html

```diff
@@ -31,15 +41,46 @@
       <div class="break"></div>
       <div class="big-number">
         <div class="big-number-value">{{ info.minutes_to_win | minutes }}</div>
-        <div class="big-number-title">Estimated Time To Win</div>
+        <div class="big-number-title">
+          Estimated Time To Win
+          <div class="infoIcon" [popper]="tooltipTimeToWin"
+               [popperTrigger]="'hover'"
+               [popperPlacement]="'bottom'"
+               [popperApplyClass]="'popperStyle'">
+               <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" viewBox="0 0 16 16">
+                <path d="M8 16A8 8 0 1 0 8 0a8 8 0 0 0 0 16zm.93-9.412-1 4.705c-.07.34.029.533.304.533.194 0 .487-.07.686-.246l-.088.416c-.287.346-.92.598-1.465.598-.703 0-1.002-.422-.808-1.319l.738-3.468c.064-.293.006-.399-.287-.47l-.451-.081.082-.381 2.29-.287zM8 5.5a1 1 0 1 1 0-2 1 1 0 0 1 0 2z"/>
```

Comment:
We've been dancing around icons for a while. I'd be okay with just installing material icons.
If this is not something you want to do, try to find a way to not repeat this in a template, for example by extracting to the component.

Reply:
Installed material icons and extracted to a template. I'm still messing around with styles so I will be a couple more days on this.